### DISCUSSION TITLE
Sony Vaio / ASUS P2B-AE

### DISCUSSION THREAD

**Discussion by MattBytesRetro** [86Box/86Box#2193] [Ideas]
I'd like to request to add the Sony Branded ASUS P2B-AE Motherboard, that was used in many Sony machines from around 1999-2000. It appears similar to other P2B boards, however it has onboard Vortex2 Audio, and the required Sony DMI code to allow for Sony OEM restores.

It's an Intel 440BX Chipset. 

Bios (BIO05UXP - Self Extracting): https://rebyte.me/ru/sony/5466/file-9027/ 

https://i.ebayimg.com/images/g/A2AAAOSwbzhhXfKL/s-l1600.jpg
https://i.ebayimg.com/images/g/vfkAAOSwPkFhpjov/s-l1600.jpg

**Comment by headsetguy37** [comment]
Fancy seeing you here!
Just so you know, there is a Feature Request option; this doesn't have to be limited to the Discussions section. Click on "New Issue", and you will have an option to request a feature.
